realm/realm-library/src/main/cpp/io_realm_mongodb_sync_SyncSession.cpp

```diff
@@ -335,7 +343,7 @@ JNIEXPORT void JNICALL Java_io_realm_mongodb_sync_SyncSession_nativeStop(JNIEnv*
         JStringAccessor local_realm_path(env, j_local_realm_path);
         auto session = app->sync_manager()->get_existing_session(local_realm_path);
         if (session) {
-            session->log_out();
+            session->close();
```

Comment:
I think this should be `pause` instead? At least that is what we use in Kotlin. We used `start/stop` as names Java but aligned the names with the other SDK's in Kotlin i.e. `pause/resume`

Reply:
Tried with `pause` but it doesn't work well provoking some tests to fail. It my be related to how we cache sessions in the SDK. 

I have noticed that `log_out` has been renamed to `force_close` so we are using it now.

Reply:
Ok, `pause` vs ` close` is also just a small performance optimization. So probably no need to figure out exactly what is causing the issue 👍

Reply:
If required we can add support for `pause` later on.